app/book_services.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+
+from app.displayer import ConsoleDisplayer, ReverseDisplayer
+from app.printer import ConsolePrinter, ReversePrinter
+from app.serializer import JSONSerializer, XMLSerializer
+
+from app.book_model import Book
+
+
+def serialize_to_json(book: Book) -> str:
+    return JSONSerializer.serialize(book)
+
+
+def serializer_to_xml(book: Book) -> str:
+    return XMLSerializer.serialize(book)
+
+
+def print_console(book: Book) -> None:
+    ConsolePrinter.print_book(book)
```

Comment:
In case where we talking about SOLID you should adhere SOLID principles, In your case, you could create separate classes for handling serialization, printing, and displaying. Also you should create separate interfaces for different types of handler which will handle somehow data related to book.

And these entities shoudl get all reldated data on main process.